.github/workflows/linting.yml

```
@@ -0,0 +1,16 @@
+name: Linting
+on: [push]
+
+jobs:
+  prose:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Vale
+      uses: errata-ai/vale-action@v1.0.1
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      with:
+        config: https://raw.githubusercontent.com/dxw/playbook/master/.vale.ini
```

```suggestion
        config: .vale.ini
```

Can't you do that, allowing changes to the Vale config on branches to run that changed config?

---

Looks like `.vale.ini` is included by default so we can loose the `config` althogether.

---

:+1: